### PR TITLE
User/saurabh dk/master

### PR DESCRIPTION
**Pull request by saurabh-dk**

### CODE REVIEW

File: lib/Task.dart [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+import 'package:path/path.dart';
+import 'package:sqflite/sqflite.dart';
+
+class Task {
+  int id;
+  String name;
+  int workDuration;
+  int breakDuration;
+  bool isComplete;
```
Comment (Feras-dev) [inline lib/Task.dart:12]: Is this where we add priority level?

**Review by tommytommychopper — APPROVED** [review]
Good ☺️

File: pubspec.lock [hunk 2]
```diff
@@ -210,6 +100,20 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "1.8.0-nullsafety.2"
+  sqflite:
```
Comment (Feras-dev) [inline pubspec.lock:103]: Is this supposed to be changed to shared preference in issue #8 ?

**Review by Feras-dev — APPROVED** [review]
Great work @saurabh-dk . Added some comments on the files to help me understand the code better. Please address when possible. Thanks.

**Comment by saurabh-dk** [comment]
Ahh yes, sure. Sorry about that.

File: pubspec.lock (hunk 2 above)
Comment (saurabh-dk) [reply pubspec.lock:103]: Yes, this is old code.

File: lib/Task.dart (hunk 1 above)
Comment (saurabh-dk) [reply lib/Task.dart:12]: Yes, not added yet.

**Comment by Feras-dev** [comment]
I think this is good to be merged to avoid blocking other's work. Any further changes should be added to the backlog as issues.

**Review by Feras-dev — APPROVED** [review]

**Review by RajathWarrier — APPROVED** [review]

**Pull request merged**